backend/spell.py

```diff
@@ -467,6 +474,58 @@ def __init__(self, artwork):
     def _validate_operations(self, board, operations):
         pass
 
+    def cast(self,board,str=[]):
+        self._validate_artwork_status(board)
+        # get linked rooms
+        populated_linked_rooms = location.linked_rooms(board, self.artwork.hex)
+        linked_room_objects = []
+        # for each room, find the objects in the room. 
+        # If there are no objects, remove the room from the list,
+        # so that only populated rooms remain; otherwise put the objects into linked_room_objects
```

Comment:
```suggestion
        # for each room, find the objects in the room. 
        # If there are no objects, remove the room from the list,
        # so that only populated rooms remain
```